docs/openapi3.yaml

```diff
@@ -2047,6 +2080,13 @@ components:
           type: number
           description: The duration of the test in seconds.
           example: 60
+        score:
+          type: number
+          description: The score of report comapring to test benchmark
+          example: 9.25
+        weights_data:
```

Comment:
consider changing the name and reference it to:
benchmark_weights

I see now there is two objects called: benchmark_weights
i would consider change the inner one to benchmark_weights_item

Reply:
i think there is only one, or i miss u?